src/common/components/vanilla-modules/button/index.js

```diff
@@ -22,7 +22,7 @@ function createButtonComponent(Component) {
     return (
       <Component {...rest} className={ className }>
         { isSpinner &&
-          <span className={ customStyle.spinner }><Spinner light/></span>
+          <IconSpinner inheritColor className={ customStyle.icon } />
```

Comment:
The spinner still changes the width of the button:

![add](https://user-images.githubusercontent.com/83575/29398586-b63b9ac8-8325-11e7-821c-e9ddb9b23d8a.gif)

To test that just go to Add repos, choose a repo and click 'Add'.

To avoid that spinner (not text) needs to be positioned absolutely over a transparent text.
That's what `<span className={ customStyle.spinner }>` wrapper over the spinner was doing before.